src/isar/models/mission.py

```diff
@@ -26,3 +26,11 @@ def __post_init__(self) -> None:
 
         if self.mission_metadata is None:
             self.mission_metadata = MissionMetadata(mission_id=self.mission_id)
+
+    def set_unique_mission_id(self) -> None:
+        plant_short_name: str = config.get("metadata", "plant_short_name")
+        robot_id: str = config.get("metadata", "eqrobot_robot_id")
+        now: datetime = datetime.utcnow()
+        self.mission_id = (
+            f"{plant_short_name.upper()}{robot_id.upper()}{now.strftime('%d%m%Y%H%M')}"
+        )
```

Comment:
Can this function also be used above in `__post_init__` instead of duplicating the code?

Reply:
Might be you have to make it static.